README.md

```diff
@@ -367,7 +384,7 @@ If your object is an instance of `UIViewController` class, perform the next step
 	}
 	```
 
-The method returns an Interaction path and a corresponding Interaction response. You can either use the `processResponse` method to let the SDK process the default response for you or process it by yourself. 
+The method returns an Interaction path and a corresponding Interaction response. You can either use the `processResponse` method to let the SDK process the default response for you or process it by yourself.
```

Comment:
"process it yourself" instead of "process it by yourself." the extra by is not needed